lib/libmbedtls/mbedtls/library/cmac.c

```diff
@@ -235,6 +235,7 @@ int mbedtls_cipher_cmac_starts( mbedtls_cipher_context_t *ctx,
         case MBEDTLS_CIPHER_AES_128_ECB:
         case MBEDTLS_CIPHER_AES_192_ECB:
         case MBEDTLS_CIPHER_AES_256_ECB:
+        case MBEDTLS_CIPHER_DES_EDE_ECB:
```

Comment:
That doesn't look right. It seems this will allow DES in CMAC mode but I think only 3DES is valid for this mode. In any case this is a change to the internals of MBedTLS, so it would need to be a separate patch ("libmbedtls: blah blah") with the OP-TEE core specific changes added on top as you did for the rest of the PR ("core: libmbedtls: add support...").

Reply:
MBEDTLS_CIPHER_DES_EDE_ECB is the name of 3DES with 128 bit keys in mbedTLS.
Yes, I know sounds strange;)

Reply:
OK thanks for clarifying, in that case this whole change looks reasonable.